Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Base
+FROM python as base
```

Comment:
You should name a more specific parent image. E.g. `python:3.9-slim-buster`
So it won't surprise you with an unwanted change.

But with slim-buster, you will need a `RUN apt-get update && apt-get install curl -y` before the curl command.